Doc/library/pathlib.rst

```diff
@@ -1068,11 +1079,10 @@ call fails (for example because the path doesn't exist).
 
 .. method:: Path.is_dir(*, follow_symlinks=True)
 
-   Return ``True`` if the path points to a directory, ``False`` if it points
-   to another kind of file.
-
-   ``False`` is also returned if the path doesn't exist or is a broken symlink;
-   other errors (such as permission errors) are propagated.
+   Return ``True`` if the path points to a directory. ``False`` will be
+   returned if the path is invalid, inaccessible or missing, or if it points
+   to something other than a directory. Use :meth:`Path.stat` to distinguish
+   between these cases.
```

Comment:
Is it necessary to clarify this?

```suggestion
   returned if the path is invalid, inaccessible or missing, or if it points
   to something else. Use :meth:`Path.stat` to distinguish between
   these cases.
```

You could still make it shorter in that case:

```suggestion
   returned if the path is invalid, inaccessible or missing, or if it doesn't
   point to a directory. Use :meth:`Path.stat` to distinguish between these
   cases.
```